frontend/packages/operator-lifecycle-manager/src/components/operator-install-page.tsx

```diff
@@ -45,7 +45,9 @@ import {
 } from '@console/shared/src/components/status/icons';
 import { InstallPlanPreview } from './install-plan';
 
-const getCRDFromObject = (obj: ClusterServiceVersionKind | InstallPlanKind | SubscriptionKind) => {
+const getInitializationResourceJSONFromCSV = (
```

Comment:
This says from `CSV` but it takes other types. Should this only take `ClusterServiceVersionKind`?

I would just call this `getInitializationResourceJSON`